openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -215,6 +218,8 @@ def run(self):
         ga = self._ga
 
         ga.elite = self.options['elitism']
+        ga.grayCode = self.options['gray']
+        ga.crossBits = self.options['cross_bits']
```

Comment:
With the coding and PEP styles we use, we don't usually use camel case for variable names or method names.  I would use gray_code and cross_bits instead.

Reply:
fixed